docs/faq.md

```diff
@@ -0,0 +1,66 @@
+| readme                  | reference                           | faq            |
+|           ---           |                ---                  |     ---        |
+| [README.md](/README.md) | [docs/reference.md](./reference.md)| this document  |
+
+# FAQ
+This is more meant as a "Fully Anticipated Questions" than a "Frequently Asked Questions", as they're anticipated as common pitfalls to using NDE.
+
+## I am new to Nix. Is this the only way to use Doom Emacs with Nix/NixOS?
+
+Nope! Doom Emacs is still perfectly usable imperatively. In fact, the very author of Doom Emacs uses NixOS and install Doom Emacs with an ["imperative" setup](https://github.com/hlissner/dotfiles/blob/master/modules/editors/emacs.nix). You could just follow the instructions on the [Doom Emacs GitHub repository](https://github.com/doomemacs/doomemacs) to get a working setup.
+
+## OK, I put your snippets into my NixOS configuration, and I put my Doom Emacs configuration as well. How do I `doom sync`?
+
+To update your Doom Emacs config, you simply rebuild your configuration. For example, in NixOS you can use `nixos-rebuild switch` (or `home-manager switch` if you use Home-Manager standalone). nix-doom-emacs will do everything else for you.
+
+In an imperative environment, Doom updates can break Emacs with no easy way to roll back.
+nix-doom-emacs moves the moving parts of your Emacs installation into the Nix build sandbox.
```

Comment:
A more correct answer would be:
> There's no `doom sync` equivalent. It just doesn't do the same thing. We build an Emacs with your Doom configuration embedded in it. `doom sync` just updates packages.

Feels like you wrote the answer before the question. Bad format? Wrong question? Maybe it shouldn't be a question. Reference material? The FAQ-Reference separation feels a bit cheesy/premature.

Maybe a reference documenting all the options and a `usage.md` documenting the pitfalls, basically this FAQ renamed, allowing for a not-so-strict format. Take the user through a tour but don't feel so forced to make everything a question. Keep the plentiful headings for easy scanning.